plugins/Jobs.cpp

```diff
@@ -103,14 +103,14 @@ bool BedrockPlugin_Jobs::peekCommand(SQLite& db, BedrockCommand& command) {
 
         // Get the list
         SQResult result;
-        const string& name = request["name"];
+        const list<string> nameList = SParseList(request["name"]);
         string operation = command.request.isSet("mockRequest") ? "IS NOT" : "IS";
         if (!db.read("SELECT 1 "
                      "FROM jobs "
                      "WHERE state in ('QUEUED', 'RUNQUEUED') "
-                     "  AND " + SCURRENT_TIMESTAMP() + ">=nextRun "
-                     "  AND name GLOB " + SQ(name) + " "
-                     "  AND JSON_EXTRACT(data, '$.mockRequest') " + operation + " NULL "
+                        "AND " + SCURRENT_TIMESTAMP() + ">=nextRun "
```

Comment:
Happy that you changed this, because it looks better. Unhappy because I will get [conflicts ](https://github.com/Expensify/Bedrock/pull/410/files#diff-d116ebb4bb9e2a3fd799045c7b182707R104) (this is just a note, no need to change anything).